runner/docker/Dockerfile.live-base

```diff
@@ -1,32 +1,29 @@
 # Based on https://github.com/huggingface/api-inference-community/blob/main/docker_images/diffusers/Dockerfile
-FROM nvidia/cuda:12.6.3-cudnn-devel-ubuntu22.04
+FROM nvidia/cuda:12.8.1-cudnn-devel-ubuntu22.04
```

Comment:
Btw, unrelated to this change, I'd love to at some point optimizing these images by using the runtime image instead of devel. I think we only really need `devel` to build ffmpeg below, so we could do a multi-stage and reduce our runtime image size by at least 2GB